dashboard/config/routes.rb

```diff
@@ -6,6 +6,8 @@ module OPS
 end
 
 Dashboard::Application.routes.draw do
+  resources :survey_results, only: [:create], defaults: { format: 'json' }
```

Comment:
without the json format here, posting the survey gives "ActionController::UnknownFormat at /survey_result" in the controller's respond_to.  Not sure whether there's a better place for it; surprised to not see this anywhere else in routes.rb.